opentelemetry-sdk/src/trace/sampler/jaeger_remote/sampling_strategy.rs

```diff
@@ -137,7 +138,13 @@ impl Inner {
             (_, _, Some(probabilistic)) => {
                 Some(Strategy::Probabilistic(probabilistic.sampling_rate))
             }
-            _ => None,
+            _ => {
+                otel_warn!(
+                    name: "Sampler.JaegerRemote.InvalidStrategy",
+                    message = "Received invalid sampling strategy from Jaeger remote endpoint. Expected one of: OperationSampling, RateLimitingSampling (max traces per second), or ProbabilisticSampling (0.0-1.0 sampling probability). No valid strategy was found in the response. Using previous strategy if available.",
```

Comment:
The message is updated as below to indicate this will use previous strategy or default sampler till a successful update.

``
                    message = "Invalid sampling strategy received from the remote endpoint. Expected one of: OperationSampling, RateLimitingSampling, or ProbabilisticSampling. Continuing to use the previous strategy or default sampler until a successful update.",
``